alerting/src/main/kotlin/org/opensearch/alerting/TriggersActionThresholdUtils.kt

```diff
@@ -40,4 +39,13 @@ object TriggersActionThresholdUtils {
     data class TriggersActionThresholdParams(
         var surplusActionCount: Int
     )
+
+    fun checkTriggerActionConfig(maxTriggerActions: Int, totalMaxTriggerActions: Int) {
+        if (maxTriggerActions > totalMaxTriggerActions) {
+            throw IllegalArgumentException(
+                "The limit number of a single trigger should not be greater than that of the overall trigger " +
+                    "$maxTriggerActions $totalMaxTriggerActions"
```

Comment:
The exception message should mention to change the setting to fix this or update the setting to increase the limit. Also the message does not mention this is a limit on actions, so we need to clarify this.

For example, this: `"The limit number of a single trigger should not be greater than that of the overall trigger $maxTriggerActions $totalMaxTriggerActions"` should become this: `"The limit number of actions for a single trigger, $maxTriggerActions, should not be greater than that of the overall max actions across all triggers of the monitor, $totalMaxTriggerActions"`